depends/md5/md5.h

```diff
@@ -23,11 +23,20 @@
 
 #include <stdint.h>
 
-struct MD5Context
+union MD5Context
 {
-    uint32_t buf[4];
-    uint32_t bits[2];
-    unsigned char in[64];
+    struct
+    {
+        unsigned char buf[16];
+        unsigned char bits[8];
+        unsigned char in[64];
+    } byte;
+    struct
+    {
+        uint32_t buf[4];
+        uint32_t bits[2];
+        uint32_t in[16];
+    } word;
 };
```

Comment:
This is Undefined Behavior™ (I think - there's some rule against accessing data in an instance of a union through more than one field, e.g. accessing byte.buf and word.buf in the same instance, although it's interesting that GCC would warn about conversions between a `char*` and `uint32_t*` but not that).